instruments.go

```diff
@@ -49,7 +48,7 @@ func newInstruments(meter metric.Meter) (*instruments, error) {
 	if instruments.latency, err = meter.Float64Histogram(
 		strings.Join([]string{namespace, "latency"}, "."),
 		instrument.WithDescription("The latency of calls in milliseconds"),
-		instrument.WithUnit(unit.Milliseconds),
```

Comment:
Any reason to change this?

Reply:
API changes. See
https://github.com/open-telemetry/opentelemetry-go/pull/3776
https://github.com/open-telemetry/opentelemetry-go/pull/3814